### PR TITLE
First release

### PR DESCRIPTION
**Pull request by santinoncs**

### CODE REVIEW

File: templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - env:
+          - name: LOG_LEVEL
+            value: debug
+          - name: REDIS_SOCKET_TYPE
+            value: tcp
+          - name: REDIS_URL
+            value: ratelimit-redis-master:6379
```
Comment (rccrdpccl) [inline templates/deployment.yaml]: Can we make it a variable, and enable/disable "packaged" redis? Also, does it make more sense having master/slave redis or would it be better sharded redis in this case?

File: templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+          - name: RUNTIME_ROOT
+            value: /data
+          - name: RUNTIME_SUBDIRECTORY
+            value: ratelimit
+          - name: USE_STATSD
+            value: "false"
+          - name: REDIS_AUTH
+            value: "LGWuol7DxF"
```
Comment (rccrdpccl) [inline templates/deployment.yaml]: :sweat_smile:

Comment (santinoncs) [reply templates/deployment.yaml]: 👎

File: templates/deployment.yaml (hunk 1 above)
Comment (santinoncs) [reply templates/deployment.yaml]: you are right, there is no persistence here. Let me fix that.

**Pull request merged**